install/amazon_rds/_03_create_iam_role.mdx

```diff
@@ -0,0 +1,31 @@
+import TabPanel, {TabTextContent} from '../../components/TabPanel'
+import CreateIAMRoleCLI from './_03_create_iam_role_cli.mdx'
+import CreateIAMRoleConsole from './_03_create_iam_role_console.mdx'
+
+export const IAMRoleCreation = () => {
+  const tabs = [
+    [ 'cli', 'CLI' ],
+    [ 'console', 'console'],
```

Comment:
I believe aws is calling the dashboard "console", you can find it in this doc too (it has "console", "AWS CLI", "AWS API"):

https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html

I first thought that "console" is not so obvious to talk about the UI dashboard thingy, but I guess it's okay if it's what aws uses?